cmd/acikubectl/cmd/debug.go

```diff
@@ -87,6 +89,41 @@ func clusterReport(cmd *cobra.Command, args []string) {
 			name: "cluster-report/logs/controller/acc.log",
 			args: accLogCmdArgs(systemNamespace),
 		},
+		{
+			name: "cluster-report/status/describe_nodes.log",
```

Comment:
Request: if can keep consistency in the name of the files. For "all describe resources" of k8s we are creating <resource>_status.log, can we do the same for the "describe nodes" ?